readme.md

```diff
@@ -131,6 +131,10 @@ Stop the spinner, change it to a yellow `⚠` and persist the current text, or `
 
 Stop the spinner, change it to a blue `ℹ` and persist the current text, or `text` if provided. Returns the instance.
 
+#### .isSpinning
+
+Returns a boolean that describes whether the instance is currently spinning.
```

Comment:
By using the `Returns` word, it looks like it's a function returning something. Should we rather use something like

> Retrieve the status of the spinner.

Short but describes what it does.

Or just leave it as is. This is really nitpicking :).

Reply:
Especially because the other functions end in parens, it's clear to me that this is a property not a function. Happy to change if you want, but if it were my choice I'd leave it. ¯\\\_(ツ)_/¯